tasks/sep-dev-0/006-holocene-sys-cfg/OVERVIEW.md

```diff
@@ -0,0 +1,170 @@
+## Tx #5: Set `OptimismPortal` storage slot in `SystemConfig`
+Sets the new `OptimismPortal` storage slot in the `SystemConfig` contract.
+
+**Function Signature:** `setAddress(bytes32,address)`
+
+**To:** `0xa6b72407e2dc9EBF84b839B69A24C88929cf20F7`
+
+**Value:** `0 WEI`
+
+**Raw Input Data:** `0xca446dd94b6c74f9e688cb39801f2112c14a8c57232a3fc5202e1444126d4bce86eb19ac00000000000000000000000076114bd29dfcc7a9892240d317e6c7c2a281ffc6`
+
+### Inputs
+**_slot:** `0x4b6c74f9e688cb39801f2112c14a8c57232a3fc5202e1444126d4bce86eb19ac`
+
+**_addr:** `0x76114bd29dFcC7a9892240D317E6c7C2A281Ffc6`
+
+
+## Tx #6: Set `OptimismMintableERC20Factory` storage slot in `SystemConfig`
+Sets the new `OptimismMintableERC20Factory` storage slot in the `SystemConfig` contract.
+
+**Function Signature:** `setAddress(bytes32,address)`
+
+**To:** `0xa6b72407e2dc9EBF84b839B69A24C88929cf20F7`
+
+**Value:** `0 WEI`
+
+**Raw Input Data:** `0xca446dd9a04c5bb938ca6fc46d95553abf0a76345ce3e722a30bf4f74928b8e7d852320c000000000000000000000000a16b8db3b5cdbaf75158f34034b0537e528e17e2`
+
+### Inputs
+**_slot:** `0xa04c5bb938ca6fc46d95553abf0a76345ce3e722a30bf4f74928b8e7d852320c`
+
+**_addr:** `0xA16b8db3b5Cdbaf75158F34034B0537e528E17e2`
+
+## Tx #7: Set `BatchInbox` storage slot in `SystemConfig`
+Sets the new `BatchInbox` storage slot in the `SystemConfig` contract.
+
+**Function Signature:** `setAddress(bytes32,address)`
+
+**To:** `0xa6b72407e2dc9EBF84b839B69A24C88929cf20F7`
+
+**Value:** `0 WEI`
+
+**Raw Input Data:** `0xca446dd971ac12829d66ee73d8d95bff50b3589745ce57edae70a3fb111a2342464dc597000000000000000000000000ff00000000000000000000000000000011155421`
+
+### Inputs
+**_slot:** `0x71ac12829d66ee73d8d95bff50b3589745ce57edae70a3fb111a2342464dc597`
+
+**_addr:** `0xff00000000000000000000000000000011155421`
+
+## Tx #8: Set `StartBlock` storage slot in `SystemConfig`
+Sets the new `StartBlock` storage slot in the `SystemConfig` contract.
+
+**Function Signature:** `setAddress(bytes32,uint256)`
+
+**To:** `0xa6b72407e2dc9EBF84b839B69A24C88929cf20F7`
+
+**Value:** `0 WEI`
+
+**Raw Input Data:** `0x8053d7d3a11ee3ab75b40e88a0105e935d17cd36c8faee0138320d776c411291bdbbb19f00000000000000000000000000000000000000000000000000000000003e1f50`
+
+### Inputs
+**_slot:** `0xa11ee3ab75b40e88a0105e935d17cd36c8faee0138320d776c411291bdbbb19f`
+
+**_block:** `4071248`
+
+## Tx #9: Set `DisputeGameFactory` storage slot in `SystemConfig`
+Sets the new `DisputeGameFactory` storage slot in the `SystemConfig` contract.
+
+**Function Signature:** `setAddress(bytes32,address)`
+
+**To:** `0xa6b72407e2dc9EBF84b839B69A24C88929cf20F7`
+
+**Value:** `0 WEI`
+
+**Raw Input Data:** `0xca446dd952322a25d9f59ea17656545543306b7aef62bc0cc53a0e65ccfa0c75b97aa9060000000000000000000000002419423c72998eb1c6c15a235de2f112f8e38eff`
+
+### Inputs
+**_slot:** `0x52322a25d9f59ea17656545543306b7aef62bc0cc53a0e65ccfa0c75b97aa906`
+
+**_addr:** `0x2419423C72998eb1c6c15A235de2f112f8E38efF`
+
+
+## Tx #10: Upgrade `SystemConfig` proxy
```

Comment:
Why are we manually setting all the slots instead of calling the `initialize` method? Is this because on sepolia/mainnet we just will use `upgrade` and not `upgradeAndCall`, so we want to preserve that exact call here and then just manually do what the initializer does to get the devnet up to date?

Reply:
It's easier than collecting all of the other configuration values that `initialize` requires. It's huge, requires the `ResourceConfig`, etc.

Doing this just allows a lot less verbosity / configuration.